spec/features/sign_in_spec.rb

```diff
@@ -0,0 +1,18 @@
+require 'rails_helper'
+
+RSpec.feature 'Sign in' do
+  let(:features) { FeaturesSpecHelper.new }
+
+  background do
+    features.sign_in(user)
```

Comment:
When you only have one line in a block, prefer the one line style: `background { sign_in(user) }`.